lib/libm/ef_rem_pio2.c

```diff
@@ -35,9 +35,9 @@
  * Table of constants for 2/pi, 396 Hex digits (476 decimal) of 2/pi
  */
 #ifdef __STDC__
-static const __int32_t two_over_pi[] = {
+static const __uint8_t two_over_pi[] = {
```

Comment:
This is fantastic. I always despaired at the size of `two_over_pi`, but never looked closely at this.